querysql/logmssql.go

```diff
@@ -0,0 +1,109 @@
+package querysql
+
+import (
+	"database/sql"
+	"encoding/hex"
+	"fmt"
+	"log"
+
+	"github.com/sirupsen/logrus"
+)
+
+func StdMSSQLLogger(logger *log.Logger) RowsLogger {
```

Comment:
It accepts the standard library logger; but not that the implementation itself uses a bit of code from logrus.  We can change the internals later, if we want to.  But having a dependency with logrus is not a problem since we need to support it anyway.